Winleafs-Server/Winleafs.Server.Api/Controllers/SpotifyController.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Threading.Tasks;
+using Winleafs.Server.Services.Exceptions;
+using Winleafs.Server.Services.Helpers;
+using Winleafs.Server.Services.Interfaces;
+
+namespace Winleafs.Server.Api.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
Please remove the "api/" else it would be "https://api.winleafs.com/api/spotify/"